migrations/2022-08-09-142755_create_tables_weather_bot/up.sql

```diff
@@ -26,9 +25,22 @@ CREATE TABLE chats (
   id BIGINT,
   user_id BYTEA,
   state client_state DEFAULT 'initial' NOT NULL, -- Initial
-  before_state client_state DEFAULT 'initial' NOT NULL, -- Initial
   selected VARCHAR(80),
   default_city_id INT,
   PRIMARY KEY (id, user_id),
   CONSTRAINT fk_cities FOREIGN KEY(default_city_id) REFERENCES cities(id)
 );
+
+CREATE TABLE forecasts (
+  chat_id BIGINT,
+  user_id BYTEA,
+  city_id INT,
+  cron_expression VARCHAR(80),
+  last_delivered_at TIMESTAMP WITH TIME ZONE,
+  next_delivery_at TIMESTAMP WITH TIME ZONE NOT NULL,
+  update_at TIMESTAMP WITH TIME ZONE NOT NULL,
+  create_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  PRIMARY KEY(chat_id, user_id, city_id),
```

Comment:
maybe let's add a separate field for primary key like you did for cities? 
(chat_id, user_id, city_id) can be added into unique key

Reply:
Done.